index.md

```diff
@@ -19,4 +19,6 @@ layout: default
   Patterns are grouped by categories (a category being a directory under
   `_posts`). Each pattern is a file under `_posts/$some_category/` (with the
   filename being `$date-$pattern_name.html`)
-3. Submit a pull request
+3. Test your changes locally with Docker Compose by running `docker-compose up`
```

Comment:
Is the markdown for the link needed? I think the url would be linked without the markdown and if not I would use angle brackets instead :thinking:

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Reply:
This is also what I thought at first, but it's not the same Markdown flavor as the one GitHub uses. It is Liquid Markdown. The only guide I found about explaining links is this one: https://gist.github.com/roachhd/779fa77e9b90fe945b0c

Reply:
This is also what I thought at first, but it doesn't seem to be the same Markdown flavor as the one GitHub uses. For some reasons, the links in plain text aren't rendered as links. Thank you for your suggestion of `<link>`, I will use it.